### PR TITLE
Pull request template

### LINKED ISSUE #271: Add rebase and line endings docs to CONTRIBUTING.md
There are two issues that are occurring regularly:
1. End of file newlines being removed (I expect this is primarily Windows users)
2. The need to rebase/refresh a pull request

We should add these to [CONTRIBUTING.md](https://github.com/WordPress/twentyseventeen/blob/master/CONTRIBUTING.md) to make these instructions easily linkable

### PR DESCRIPTION
**Pull request by josephfusco**
Adds a pull request template to the GitHub project.

See GitHub article: https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/

This doesn't fix #271 but supports it.

### CODE REVIEW

**Comment by samikeijonen** [comment]
+1. I'd also need helpful text how to sync your local master branch with remote branch. And how to use rebase for custom PR branch after that. Or is that even suggested workflow.

**Comment by grappler** [comment]
@samikeijonen See this https://github.com/WordPress/twentyseventeen/pull/90#issuecomment-250616393

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+##### Screenshots (if appropriate)
+
+##### Questions:
+Please make sure all the below checkboxes have been checked before submitting your PR.
+
+- [ ] Are all the above sections filled out appropriately?
+- [ ] Have you read [CONTRIBUTING.md](https://github.com/WordPress/twentyseventeen/blob/master/CONTRIBUTING.md)?
+- [ ] Did you maintain a new line at the end of each file modified/added?
+- [ ] If this adds new js dependencies, did you add them to [.jscrc](https://github.com/josephfusco/twentyseventeen/blob/master/.jscsrc) & [.jshintignore](https://github.com/WordPress/twentyseventeen/blob/master/.jshintignore)?
```
Comment (enodekciw) [inline .github/PULL_REQUEST_TEMPLATE]: Both [.jscsrc] and [.jsintignore] should be linked to WordPress/twentyseventeen

Comment (josephfusco) [reply .github/PULL_REQUEST_TEMPLATE]: @enodekciw good catch 👍

**Comment by samikeijonen** [comment]
Thanks @grappler for tip. First step (syncing master branch from this master branch) is crystal clear.

But I'm confused of command `git reset --hard master` after checking out on my `svg` branch. Wouldn't that make my `svg` branch exactly the same as `master` branch. And I would lose all the modifications I made on `svg` branch.

Shouldn't I use `git rebase master` command instead and try to solve merge conflicts manually?

**Comment by mor10** [comment]
@samikeijonen When I work on non-WP projects, we typically use a process like the one outlined in this article:  https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

I have no idea if this is what the team is looking for, but it's a pretty clean and straight-forward process:

```
// Checkout master:
git checkout master
// Get the updates:
git pull --rebase upstream master
// Checkout your working feature branch:
git checkout [working branch]
// Rebase updates to the branch:
git rebase master
// resolve and commit conflicts (happens in editor context)
// Force the changes to the remote branch (pull request):
git push --force
```

If this is not a desirable methodology, please let me know.

**Comment by samikeijonen** [comment]
Thanks @mor10. That's pretty much the workflow I'm using also. But instead of `git pull --rebase upstream master` I sync the master branch like [outlined in this article](https://help.github.com/articles/syncing-a-fork/).

In #206 I probably used `merge` and messed PR. I'll try `rebase` this time in #253. Fingers crossed.

**Pull request merged**